neevaco/neeva-ios
# Add "Search with Neeva" to Shortcuts and Siri Suggestions

- Edit Search with Neeva shortcut item (the display strings)
- When we submit text to the url bar
  - add the shortcut to shortcut suggestions (can be viewed in Shortcuts app under Gallery)
  - donate the interaction (can be viewed by turning on "Display Recent Shortcuts" and "Display Donations on Lock Screen" in your phone's developer's settings)

@darinwf  we could put the search text in the donated object, which will open the same search in a new tab when it is opened; my understanding is that our goal is to open the app on a new tab with this suggestion.

putting this out now to get some feedback

## Checklists

### How was this tested?

### Manual test cases
Do a search, verify that a short cut suggestion is created, and that a siri suggestion was donated.
Open the shortcut, verify that it opens a blank search.
Open the suggestion, verify that it opens a blank search.

With the shortcut, provide a search text, run shortcut, verify that it opens a new search with that query

### Screenshots and screen recordings

## Issues addressed
